lang/en/docs/materials-designer/header-menu/advanced/jupyterlite-dialog.md

```diff
@@ -0,0 +1,70 @@
+# JupyterLite Transformation Dialog
+
+The JupyterLite Transformation Dialog enables the modification of materials within the JupyterLite environment, which is a lightweight implementation of JupyterLab that runs entirely in the browser. This dialog is accessible via the ["Advanced" menu](../advanced.md) within the Materials Designer interface.
+
+![Materials Designer header with Advanced menu opened](../../../../../../images/materials-designer/jupyterlite_dialog/open-jupyterlite-dialog.webp "Open JupyterLite Transformation Dialog")
```

Comment:
Let's add a link to the lang/en/docs folder and link images into it to avoid having to go all the way to the top level here

Reply:
Alternatively, we may use absolute paths for images, in this case, it's `/images/materials-designer/jupyterlite_dialog/open-jupyterlite-dialog.webp`.